cmd/azbrowse/main.go

```diff
@@ -78,8 +78,20 @@ func run(settings *Settings) {
 	// Close the span used to track startup times
 	span.Finish()
 
+	// recover from panic if one occurred leaving terminal usable
+	defer func() {
+		if r := recover(); r != nil {
+			g.Close()
+			fmt.Printf("\n\nA crash occurred: %s \n", r)
+			debug.PrintStack()
+			fmt.Println("Please visit https://github.com/lawrencegripper/azbrowse/issues to raise a bug.")
+			os.Exit(1)
+		}
+	}()
+
 	// Start the main loop of gocui to draw the UI
 	if err := g.MainLoop(); err != nil && err != gocui.ErrQuit {
+		g.Close()
```

Comment:
Is this needed? The line below will always throw a panic that'll be caught by the deferred recover statement which will in turn call `g.Close()` - is there another reason for it?

Reply:
Good call, removed.